### PR TITLE
Fixed Async Discrepancies, standarized async pulls.

### PR DESCRIPTION
**Pull request by rauchp**
ok but this time nav drawer ~actually~ updates when we change a pic!

### CODE REVIEW

File: civilapp/current/ListenerView.js [hunk 1]
```diff
@@ -235,7 +235,7 @@ class ListenerView extends React.Component {
     }
     //Check if already followed, then unfollow instead
     const userInfo = this.state.currentUser
-    const userPreview = {
+    const userInfo = {
```
Comment (EphemeralX) [inline civilapp/current/ListenerView.js]: Use let for the above userInfo or call this userPreview.

File: civilapp/current/ListenerView.js [hunk 2]
```diff
@@ -299,14 +299,14 @@ class ListenerView extends React.Component {
   leaveBroadcast = async () => {
     AgoraAPI.leaveChannel()
     const userInfo = this.state.currentUser
-    const userPreview = {
+    const userInfo = {
```
Comment (EphemeralX) [inline civilapp/current/ListenerView.js]: userPreview

File: civilapp/profile/ProfileView.js [hunk 3]
```diff
@@ -66,19 +66,19 @@ class ProfileView extends React.Component {
 
   componentDidMount = async () => {
     // Gets User ID of logged in user.
-    let userPreview = await AsyncStorage.getItem('USER_PREVIEW');
-    userPreview = JSON.parse(userPreview);
+    let thisUserInfo = await AsyncStorage.getItem('USER_INFO');
```
Comment (EphemeralX) [inline civilapp/profile/ProfileView.js]: call this loggedInUserInfo or authenticatedUserInfo

File: civilapp/profile/ProfileView.js [hunk 4]
```diff
@@ -123,9 +123,16 @@ class ProfileView extends React.Component {
             input: userDetails
           })
         )
+        const newUserInfo = {
+          id: userId,
+          username: username,
+          profileImageUrl: image.uri
+
```
Comment (EphemeralX) [inline civilapp/profile/ProfileView.js]: nit

File: civilapp/current/ListenerView.js (hunk 2 above)
Comment (rauchp) [reply civilapp/current/ListenerView.js]: >listenerview

just noticed that

**Pull request merged**